app/src/main/java/org/oppia/android/app/viewmodel/BUILD.bazel

```diff
@@ -16,10 +17,28 @@ kt_android_library(
     srcs = [
         "ObservableViewModel.kt",
     ],
+    visibility = ["//visibility:public"],
     deps = [
         artifact("androidx.databinding:databinding-adapters"),
         artifact("androidx.lifecycle:lifecycle-extensions:2.2.0"),
         artifact("androidx.lifecycle:lifecycle-livedata-core:2.2.0"),
```

Comment:
Let's remove 

```
        artifact("androidx.lifecycle:lifecycle-livedata-core:2.2.0"),
        artifact("androidx.lifecycle:lifecycle-livedata-ktx:2.2.0"),
```

from this set of dependencies. We want to only include the dependencies we use in the target.

Reply:
Done